php/php.editor/src/org/netbeans/modules/php/editor/actions/ImportDataCreator.java

```diff
@@ -87,6 +88,10 @@ public ImportData create() {
     }
 
     private void processFQElementName(final String fqElementName) {
+        // GH-6039: avoid getting all types
+        if (!StringUtils.hasText(fqElementName)) {
+            return;
+        }
         // GH-6075
         String fqeName = CodeUtils.removeNullableTypePrefix(fqElementName);
         Collection<FullyQualifiedElement> possibleFQElements = fetchPossibleFQElements(fqeName);
```

Comment:
Before: got all types with an empty type name here. So, the following method was called many times.

https://github.com/apache/netbeans/blob/9e68d3d9630f9403fc14c3a389c803643d92521f/php/php.editor/src/org/netbeans/modules/php/editor/actions/FixDuplicateImportStmts.java#L248-L259